src/dotnet/CarbonAware.WebApi/README.md

```diff
@@ -77,4 +77,15 @@ EG
   "embodiedEmissionsValue": null,
   "functionalUnitValue": null
 }
-```
\ No newline at end of file
+```
+
+
+## Error Handling
```

Comment:
I think it'd be useful to add an extra sentence here basically along the lines of "if you throw an exception somewhere in the stack but catch it before it hits the controller, then this won't be triggered. If the exception hits the controller than this standardized process takes over."